src/components/periodic-report/periodic-report.resolver.ts

```diff
@@ -22,4 +31,25 @@ export class PeriodicReportResolver {
     );
     return reportFile;
   }
+
+  @ResolveField(() => SecuredFile, {
+    nullable: true,
+  })
```

Comment:
Secured* objects are not nullable. canRead/Edit are always given.
```suggestion
  @ResolveField(() => SecuredFile)
```